lib/consult/template_functions.rb

```diff
@@ -30,7 +30,7 @@ def query_nodes(*args)
       query(*args)&.Nodes&.map { |node| node['Node'] }
     end
 
-    def key(key, options: nil, not_found: :reject, found: :return)
+    def key(key, options: {}, not_found: :reject, found: :return)
```

Comment:
Basically this is the same issue for the method key, this is the issue fixed:
```
NoMethodError:
  undefined method `[]' for nil:NilClass
    options[:recurse] ? ['recurse'] : []
```
The issue was present in the following file:
```
diplomat-2.6.4/lib/diplomat/kv.rb
```